pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -77,6 +77,13 @@ type Manager interface {
 	// and is consulted to achieve NUMA aware resource alignment among this
 	// and other resource controllers.
 	GetTopologyHints(*v1.Pod, *v1.Container) map[string][]topologymanager.TopologyHint
+
+	// GetCPUs implements the podresources.CPUsProvider interface to provide allocated
+	// cpus for the container
+	GetCPUs(podUID, containerName string) []int64
+
+	// GetAllCPUs provides the ordered IDs of all the online CPUs found on the node.
```

Comment:
its not really all right?  its just the list of cpus that are schedulable for the node, right?

Reply:
would expect this to match language here:

	// GetAssignableCPUs returns the assignable (not allocated) CPUs

Reply:
yes, stale comment, like you said. Will update. Thanks for catching this!